frontend/sw360-portlet/src/main/resources/portal.properties

```diff
@@ -10,4 +10,13 @@
 #auto.login.hooks=org.eclipse.sw360.portal.users.TestAutoLogin
 #auto.login.hooks=org.eclipse.sw360.portal.users.SSOAutoLogin
 login.events.post=org.eclipse.sw360.portal.users.LoginAction
-servlet.service.events.pre=org.eclipse.sw360.portal.hooks.BuildInfoForVelocityProviderHook
\ No newline at end of file
+servlet.service.events.pre=org.eclipse.sw360.portal.hooks.BuildInfoForVelocityProviderHook
+
+
+# Specify the number of minutes before a session expires. This value is
+# always overridden by the value set in web.xml.
+session.timeout=30
```

Comment:
Why do we need to set the session timeout in this specific file (portal.properties)?

Currently in our liferay configuration the session timeout is already configured.
This timeout is configured in the web.xml (ROOT/WEB-INF) 
```
	<session-config>
		<session-timeout>30</session-timeout>
	</session-config>
```

Which configuration file is now the "master"?
- web.xml or portal.properties?